### PR TITLE
DRIVERS-1603: Exclude mapReduce from serverless testing

### PR DESCRIPTION
**Pull request by MikalaiMazurenka**
[DRIVERS-1603](https://jira.mongodb.org/browse/DRIVERS-1603)

`mapReduce` operation is not intended to work on serverless (see comment from [CLOUDP-93419](https://jira.mongodb.org/browse/CLOUDP-93419))

### CODE REVIEW

**Comment by jmikola** [comment]
@MikalaiMazurenka: having mongodb/dbx-spec-owners-retryability as a reviewer was correct. There is no need to tag individual spec owners directly -- I'm not sure if that interferes with GitHub's calculation of a required spec owner LGTM.

**Review by jmikola — APPROVED** [review]
Deferring to @patrickfreed for a second review.

**Comment by alcaeus** [comment]
> @MikalaiMazurenka: having mongodb/dbx-spec-owners-retryability as a reviewer was correct. There is no need to tag individual spec owners directly -- I'm not sure if that interferes with GitHub's calculation of a required spec owner LGTM.

Unfortunately, this is how the GitHub UI shows the process of the dbx-spec-owners-* groups using round robin review assignments: creating the PR requests a review from the group, directly after which the group removes itself and adds one of its members as required reviewer. Unfortunately, this is shown as being done by the author and hides the fact this was done automatically. I only noticed that a couple of days ago when I saw this in action in my own pull request.

**Review by patrickfreed — APPROVED** [review]

**Pull request merged**
